src/role_squadheal.js

```diff
@@ -34,26 +34,26 @@ roles.squadheal.preMove = function(creep, directions) {
     creep.memory.routing.reverse = false;
     return false;
   }
-  
+
```

Comment:
Nice catch, thanks for looking into it.

If you like to get in contact either via gitter (screeps-bot-tooangel) or screeps slack.